lib/vehicle.rb

```diff
@@ -22,4 +26,13 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def set_platetype(type)
+    raise TypeError, "plate_type must be a symbol" unless type.is_a?(Symbol)
+    @plate_type = type
+  end
+
+  def set_registration_date
+    @registration_date = Date.new(Date.today.year, Date.today.month, Date.today.day)
```

Comment:
Also could just do `@registration_date = Date.today` :) Nice job making a method for this attribute change!